### DISCUSSION TITLE
Dependabot || Dependencies in general

### DISCUSSION THREAD

**Discussion by TfedUD** [PH-Tools/honeybee_ph#20] [General]
@PH-Tools soo the unit tests are failing coz dependabot PR's are messing up the stuff on the repo's requirements.txt vs the same stuff on the LBT dependencies requirements such as:

![image](https://user-images.githubusercontent.com/63176386/172927552-ef62853d-1727-4742-90d5-42e782c3100f.png)

SO I think culling redundant dependencies may be solution for that? i.e; should prolly only really get dependabot PR`s having to do with LBT dependencies.
Thoughts?

**Comment by ed-p-may** [comment]
or is there a way to do > rather than == for versions? Maybe then it would
not matter so much?

**Reply by TfedUD** [reply]
yeah for sure: I'm looking into the config settings right now to see how to make it do that; figured that would be less invasive as well as make a touch more sense

**Reply by TfedUD** [reply]
@PH-Tools CM's $0.02:
```Given that honeybee-ph depends upon honeybee, which depends upon ladybug, which depends upon click==7.1.2, you will always get click==7.1.2 when you add the honeybee dependency. So, yes, you should remove the redundant dependencies.```

**Comment by TfedUD** [comment]
@PH-Tools I fixed the thing but I still cant get the coverage action to pass on the PH tools repo; though it's passing on mine. so I temporarily removed the coverage action from main while I troubleshoot on the other PR
## [Refer to PR #24](https://github.com/PH-Tools/honeybee_ph/pull/24)
